cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -2491,7 +2617,8 @@ display.page.search.mode = fast
 request.ui_dispatch_app = cyences_app_for_splunk
 request.ui_dispatch_view = search
 search = `cs_crowdstrike_eventstream` "metadata.eventType"=DetectionSummaryEvent action="allowed" \
```

Comment:
Let's keep both allowed and not allowed.
But for allowed keep the severity as it is. For not-allowed keep the severity as info.